automation/test.sh

```diff
@@ -274,7 +274,7 @@ export IMAGE_PREFIX_ALT=${IMAGE_PREFIX_ALT:-kv-}
 
 build_images
 
-trap '{ collect_debug_logs; echo "Dump kubevirt state:"; make dump; }' ERR
+trap '{ collect_debug_logs; }' ERR
```

Comment:
Don't we want to move it below?

Reply:
There are `make dump`s further below in the script.

Reply:
`make cluster-sync` has its own error handling - https://github.com/kubevirt/kubevirt/blob/ee0144cfd43170ac12bf973867c93850f78697d5/hack/cluster-deploy.sh#L72